README.md

```diff
@@ -43,4 +43,16 @@ For common questions about the APIs, see [Frequently Asked Questions](faq.adoc).
 Apply for API credentials on [Zettle Developer Portal](https://developer.zettle.com/register).
 
 ## Get help
-Contact our [Integrations team](mailto:api@zettle.com) for more information. 
\ No newline at end of file
+Contact our [Integrations team](mailto:api@zettle.com) for more information. 
+
+When contacting the Integrations team, please provide the following information:
+
+* Description about the issue. That includes:
+    * What's the integration used for?
+    * Which API or APIs are you using?
+    * What went wrong? 
+
+* Information about affected merchants. That includes:
+    * Emails of affected merchants.
```

Comment:
super minor, but maybe add an `or` (email or UUID, I don't think we need both.)

Reply:
> LGTM!
> However, its good to wait for Dejan's feedback @dmilovanovi

Thank you! :) I only asked Dejan to check before I created the PR but forgot to add him in the PR... I will ask him to check whether email is optional.